rclcpp_components/src/component_manager.cpp

```diff
@@ -39,10 +39,12 @@ ComponentManager::ComponentManager(
 {
   loadNode_srv_ = create_service<LoadNode>(
     "~/_container/load_node",
-    std::bind(&ComponentManager::on_load_node, this, _1, _2, _3));
+    std::bind(&ComponentManager::on_load_node, this, _1, _2, _3),
+    rclcpp::QoS(200));
```

Comment:
Should this be instead `rclcpp::ServiceQoS().keep_last(200)` ?

Reply:
I think they are both same:

https://github.com/ros2/rclcpp/blob/0f331f90a99da40f7b7a69b4f55b30b7245d294f/rclcpp/include/rclcpp/qos.hpp#L137-L146

I could also do: `rclcpp::QoS(rclcpp::KeepLast(history_depth))`, is there any preference for using the more explicit calls?

Reply:
If the defaults for services ever change (e.g. reliability, livelines, etc.), using `rclcpp::ServicesQoS` would avoid any issues in the future, so I'd prefer `rclcpp::ServicesQoS().keep_last(200)`

Reply:
Yeah, agreed with that.

Reply:
Applied: https://github.com/ros2/rclcpp/pull/2363/commits/9bf1b0923e38561433c599661753232df4d8c4b2